www/class/centreon-clapi/centreonHost.class.php

```diff
@@ -553,10 +556,9 @@ public function getparam($parameters = null)
                             $ret = $ret[$field];
                             break;
                     }
-                    if (!$resultString) {
-                        $resultString = $ret;
-                    } else {
-                        $resultString = $resultString . $this->delim . $ret;
+                    if ($exportedFields[$paramSearch] == 0) {
```

Comment:
if $resultString is not define you will take a notice, define before the foreach.

Reply:
```suggestion
                    if (!isset($exportedFields[$paramSearch])) {
```

Reply:
to avoid PHP notice on an undefined key of the array